Makefile

```diff
@@ -139,6 +142,9 @@ fmt: install-tools
 fmt-sh: install-tools
 	${SHFMT} -w -d -i 5 .
 
+linter: install-tools
+	${LINTER} run ./...
```

Comment:
why are we not using `golint` as described here: https://sparkbox.com/foundry/go_vet_gofmt_golint_to_code_check_in_Go

Reply:
This includes `go-lint` but it [disabled by default](https://golangci-lint.run/usage/linters/#disabled-by-default)